src/schemas/json/hugo.json

```diff
@@ -0,0 +1,544 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$comment": "https://gohugo.io/getting-started/configuration/",
+  "title": "Hugo static site generator config file schema",
+  "type": "object",
+  "properties": {
+    "archetypeDir": {
+      "description": "The directory where archetype files are stored\nhttps://gohugo.io/getting-started/configuration/#archetypedir",
+      "type": "string",
+      "default": "archetypes",
+      "minLength": 1
+    },
+    "assetDir": {
+      "description": "The directory where asset files are stored\nhttps://gohugo.io/getting-started/configuration/#assetdir",
+      "type": "string",
+      "default": "archetypes",
```

Comment:
Shouldn't that be
`"default": "assets"`?

Reply:
Yes, it's my mistake.